lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -3997,6 +3997,7 @@ void ByteCodeGenerator::StartEmitFunction(ParseNode *pnodeFnc)
                 !funcInfo->Escapes() &&
                 funcInfo->frameObjRegister != Js::Constants::NoRegister &&
                 !ApplyEnclosesArgs(pnodeFnc, this) &&
+                !pnodeFnc->sxFnc.HasDefaultArguments() &&
```

Comment:
Yes. This specifically has to do with lookups from within the default expression.